dashboard/templates/view_file_access_within_week.html

```diff
@@ -19,33 +19,31 @@
 {% endblock %}
 {% block content %}
     <!-- week slider -->
-    <br/>
-    <br/>
-    <h2>What Students Are Reading?</h2>
+    <h4 align="center">What Students Are Reading</h4>
 
 
     <input type="text" id="slider" class="slider">
 
-
     <input type="hidden" name="slider_num" id="slider_num" />
     <div/>
-
-    Top files access by
+    <br/>
+    <h6 align="center">Files accessed by
```

Comment:
Minor but these align attribute should be using bootstrap class="text-center" as align is deprecated https://getbootstrap.com/docs/3.3/css/#type-alignment

Reply:
updated!